### ISSUE TITLE
HmIP-RC8: No keypress events

### ISSUE THREAD

**Issue by akloeckner** [danielperna84/pyhomematic#317]
**Home Assistant version (if applicable):**
0.108.3.

**Problem-relevant `configuration.yaml` entries and steps to reproduce:**
```yaml
automation:
- alias: Fernbedienung HomeMatic
  id: fernbedienunghm
  trigger:
  - platform: event
    event_type: homematic.keypress
    event_data:
      name: "Fernbedienung 1"
      param: PRESS_SHORT
```

Here's the debug log when I push a button the remote
```
Jul 26 16:27:32 raspberrypi hass[22912]: 2020-07-26 16:27:32 DEBUG (Thread-3) [pyhomematic._hm] RPCFunctions.event: interface_id = homeassistant-ip, address = 000B1BE98D94A5:0, value_key = CONFIG_PENDING, value = False
Jul 26 16:27:32 raspberrypi hass[22912]: 2020-07-26 16:27:32 INFO (Thread-3) [pyhomematic.devicetypes.generic] HMGeneric.event: address=000B1BE98D94A5:0, interface_id=homeassistant-ip, key=CONFIG_PENDING, value=False
Jul 26 16:27:32 raspberrypi hass[22912]: 2020-07-26 16:27:32 DEBUG (Thread-3) [pyhomematic.devicetypes.generic] HMGeneric.event: Using callback functools.partial(<function _hm_event_handler at 0x69a1bb70>, <homeassistant.core.HomeAssistant object at 0x753a54d0>, 'ip')
Jul 26 16:27:32 raspberrypi hass[22912]: 2020-07-26 16:27:32 DEBUG (Thread-3) [pyhomematic._hm] RPCFunctions.event: interface_id = homeassistant-ip, address = 000B1BE98D94A5:0, value_key = UNREACH, value = False
Jul 26 16:27:32 raspberrypi hass[22912]: 2020-07-26 16:27:32 INFO (Thread-3) [pyhomematic.devicetypes.generic] HMGeneric.event: address=000B1BE98D94A5:0, interface_id=homeassistant-ip, key=UNREACH, value=False
Jul 26 16:27:32 raspberrypi hass[22912]: 2020-07-26 16:27:32 DEBUG (Thread-3) [pyhomematic.devicetypes.generic] HMGeneric.event: Using callback functools.partial(<function _hm_event_handler at 0x69a1bb70>, <homeassistant.core.HomeAssistant object at 0x753a54d0>, 'ip')
Jul 26 16:27:32 raspberrypi hass[22912]: 2020-07-26 16:27:32 DEBUG (Thread-3) [pyhomematic._hm] RPCFunctions.event: interface_id = homeassistant-ip, address = 000B1BE98D94A5:0, value_key = RSSI_DEVICE, value = -59
Jul 26 16:27:32 raspberrypi hass[22912]: 2020-07-26 16:27:32 INFO (Thread-3) [pyhomematic.devicetypes.generic] HMGeneric.event: address=000B1BE98D94A5:0, interface_id=homeassistant-ip, key=RSSI_DEVICE, value=-59
Jul 26 16:27:32 raspberrypi hass[22912]: 2020-07-26 16:27:32 DEBUG (Thread-3) [pyhomematic.devicetypes.generic] HMGeneric.event: Using callback functools.partial(<function _hm_event_handler at 0x69a1bb70>, <homeassistant.core.HomeAssistant object at 0x753a54d0>, 'ip')
```

**Describe the bug**
The HmIP-RC 8 does not generate `homematic.keypress` events in Home Assistant.

**To Reproduce**
* Connect remote to CCU
* Listen for `homematic.keypress` events in Home Assistant.
* Push button
* Observe nothing happening

**Expected behavior**
An event should be fired.

**Screenshots**
N/A

**Additional context**
I have a non-IP Home Matic 8 Button remote `HM-RC-8`, which works just fine.

**Comment by akloeckner** [comment]
So, I found the fix myself. It's **not** on the pyhomematic side (nor in home assistant). 

TL;DR: The CCU(2) does not receive encrypted telegrams it does not subscribe for. So, either encryption has to be turned off. Or you need to create a dummy script for each button and transfer it to the remote control. You can delete the dummy script afterwards.

See here for a longer explanation (in German):
https://www.symcon.de/wiki/homematic-taster_in_ips/

**Comment by danielperna84** [comment]
That's already in the documentation [here](https://www.home-assistant.io/integrations/homematic/#homematickeypress-events-for-homematic-ip-devices) BTW. 😉

**Comment by akloeckner** [comment]
:man_facepalming: I could have thought of that... :+1:
